nx-dev/ui-common/src/lib/footer.tsx

```diff
@@ -5,61 +5,43 @@ import { DiscordIcon } from './discord-icon';
 
 export function Footer(): JSX.Element {
   const navigation = {
+    nx: [
+      { name: 'Status', href: 'https://status.nx.app' },
+      { name: 'Security', href: 'https://security.nx.app' },
+    ],
+    nxCloud: [
+      { name: 'App', href: 'https://cloud.nx.app' },
+      { name: 'Docs', href: '/ci' },
```

Comment:
Can we maybe directly link to the CI section: /ci/intro/ci-with-nx

nx.dev/ci should redirect to it, but when trying it on the live preview here it brought me right into the index page here 
<img width="1113" alt="image" src="https://github.com/user-attachments/assets/67f3e397-f458-47b7-9e3a-d41cefb86369">